docs/csdk-guide.md

```diff
@@ -68,20 +68,42 @@ JNIEnv *initJVM() {
 bool readFromLocalWithoutProjection(JNIEnv *env) {
 
     CarbonReader carbonReaderClass;
-    carbonReaderClass.builder(env, "../resources/carbondata", "test");
+    carbonReaderClass.builder(env, "../resources/carbondata");
```

Comment:
Already main.cpp has these same examples. Please give a link to that file and remove from here. 
Because if any future change happens, we need not have to change at two places and keep duplicate code samples.

Reply:
SDK guide also has this problem, do we need remove it too?